fixtures/stairville/quad-par-profile5x8w-rgbw.json

```
@@ -0,0 +1,255 @@
+          "type": "Effect",
+          "effectName": "Auto Show 7"
+        },
+        {
+          "dmxRange": [175, 199],
+          "type": "Effect",
+          "effectName": "Auto Show 8"
+        },
+        {
+          "dmxRange": [200, 224],
+          "type": "Effect",
+          "effectName": "Auto Show 9"
+        },
+        {
+          "dmxRange": [225, 249],
+          "type": "Effect",
+          "effectName": "Auto Show 10"
+        },
+        {
+          "dmxRange": [250, 255],
+          "type": "Effect",
+          "effectName": "Music Show",
+          "soundControlled": true
+        }
+      ]
+    },
+    "Color": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 18],
+          "type": "ColorPreset",
+          "comment": "Dark",
+          "colors": ["#000000"]
+        },
+        {
+          "dmxRange": [19, 37],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [38, 56],
+          "type": "ColorPreset",
+          "comment": "Orange",
+          "colors": ["#ff8800"]
+        },
+        {
+          "dmxRange": [57, 75],
+          "type": "ColorPreset",
+          "comment": "Yellow",
+          "colors": ["#ffff00"]
+        },
+        {
+          "dmxRange": [76, 94],
+          "type": "ColorPreset",
+          "comment": "Light Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [95, 113],
+          "type": "ColorPreset",
+          "comment": "Light Blue",
+          "colors": ["#00bbff"]
+        },
+        {
+          "dmxRange": [114, 132],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [133, 151],
+          "type": "ColorPreset",
+          "comment": "Violet",
+          "colors": ["#ad03fc"]
+        },
+        {
+          "dmxRange": [152, 170],
+          "type": "ColorPreset",
+          "comment": "Pink",
+          "colors": ["#fc03d7"]
+        },
+        {
+          "dmxRange": [171, 189],
+          "type": "ColorPreset",
+          "comment": "DarkWhite",
+          "colors": ["#ffffff"]
+        },
+        {
+          "dmxRange": [190, 208],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [209, 227],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [228, 246],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [247, 255],
+          "type": "ColorPreset",
+          "comment": "Red + Green + Blue + White",
+          "colors": ["#ff0000", "#00ff00", "#0000ff", "#ffffff"]
+        }
+      ]
+    }
+  },
+  "modes": [
+    {
+      "name": "4ch",
```

```suggestion
      "name": "4-channel",
      "shortName": "4ch",
```

Please also fix this for the other modes.